.gitignore

```diff
@@ -12,3 +12,4 @@ cli/badgeyay.egg-info
 cli/build/
 cli/dist/
 .vscode/
+badgeyay/
```

Comment:
This is not needed as it's user wish to specify the `virtualenv` folder and it can vary